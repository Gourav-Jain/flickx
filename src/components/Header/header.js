import React from 'react'
import { Container, Segment, Header } from 'semantic-ui-react'
import "../../index.css";
export const HeaderBar = () => (
    <Segment inverted color="black">
        <Container>
            <Header as="h1" textAlign='center' inverted >
                Flick X
            </Header>
        </Container>
    </Segment >
)

export default HeaderBar;
