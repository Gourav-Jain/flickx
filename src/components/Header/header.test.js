import React from "react";
import { shallow } from "enzyme";

import Header from "./header";

describe("<Header />", () => {

    beforeEach(jest.clearAllMocks);

    describe("@actions", () => {
        test("should display header", () => {
            const header = shallow(<Header />);
            expect(header).toMatchSnapshot();
        });
    });
});