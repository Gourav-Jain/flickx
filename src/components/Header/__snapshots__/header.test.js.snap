// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> @actions should display header 1`] = `ShallowWrapper {}`;
